client/package.json

```diff
@@ -7,6 +7,7 @@
     "@reach/router": "^1.2.1",
     "@testing-library/react": "^8.0.1",
     "geolib": "^3.0.4",
+    "date-fns": "^1.30.1",
```

Comment:
What is this package for?

Reply:
This package is for to work with dates, `date-fns` is an awesome library, you can read more [here](https://date-fns.org/).